server/apps/publish/publish_filters/filter_condition.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import logging
+import re
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+
+logger = logging.getLogger(__name__)
+
+
+class FilterConditionResource(Resource):
+    schema = {
```

Comment:
Yes, It may be needed. I didn't have much chance to think how to keep the filter conditions unique as we want to use them with different combinations in publish filters. Name -may- provide some help, say if a user creates a filter_condition: anpa_category_id in [4] and calls it 'sport' stories this may be used in the filter. 
Problems:
- if someone else uses the same name with a different filter condition
- if someone uses the same filter condition with a different name